gapii/cc/pack_string_writer.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef GAPII_PACK_STRING_WRITER_H
+#define GAPII_PACK_STRING_WRITER_H
+
+#include "core/cc/string_writer.h"
+
+namespace gapii {
+
+// PackStringWriter is used to write 'Pack' strings to a core::StreamWriter.
+class PackStringWriter : public core::StringWriter {
```

Comment:
Might be clearer to call this `ChunkWriter` given that it writes Chunks [as described here](https://github.com/google/gapid/tree/master/core/data/pack#chunkt).